examples/ConnectNoEncryption/ConnectNoEncryption.ino

```diff
@@ -91,61 +92,92 @@ void printWifiData() {
   Serial.print("IP Address: ");
   Serial.println(ip);
 
-  // print your MAC address:
+  // print the MAC address of your WiFi module:
   byte mac[6];
   WiFi.macAddress(mac);
   Serial.print("MAC address: ");
-  Serial.print(mac[5], HEX);
-  Serial.print(":");
-  Serial.print(mac[4], HEX);
-  Serial.print(":");
-  Serial.print(mac[3], HEX);
-  Serial.print(":");
-  Serial.print(mac[2], HEX);
-  Serial.print(":");
-  Serial.print(mac[1], HEX);
-  Serial.print(":");
-  Serial.println(mac[0], HEX);
+  for (uint8_t i = 0; i < 6; i++) {
```

Comment:
Before this patch MAC is printed from mac[5] to mac[0], 
now it's printed from 0 to 5 - is it intended ?

Reply:
yes, it was inverted before. I've mentioned it in my first comment of this PR.